docs/design/ovs-pipeline.md

```diff
@@ -586,6 +583,23 @@ is required), as well as for local Pod-to-Pod traffic.
 table=70, priority=0 actions=goto_table:80
 ```
 
+### L3DecTTLTable (71)
+This is the table to decrement TTL for L3 traffic on the source Node. It implements the following functionality:
+   
+ * TTL is decremented by one on the packet which is sent from a local Pod and destined
+   for a remote Pod. But for the traffic that enters OVS pipeline from the local gateway
+   and destined for a remote Pod, TTL decrement is not taken on the source Node. TTL is
```

Comment:
"is destined"
Probably "TTL is not decremented in OVS on the source Node", since TTL can be incremented by the IP stack before OVS?

Reply:
Do you mean TTL can be decremented by the IP stack before OVS?  I think there should include two cases: 1) the actual client is Pod, and kube-proxy is used to do DNAT for Service. In this case, TTL should be decremented by IP stack; 2) the actual client is from the gw0 (including the cases from Node to remote Pod, or from gw0 to remote Pod), such case we should not decrement TTL, for the gw0 and remote gw0 are "connected on the same router" directly(on-link)